blocks/magazine-summary/magazine-summary.js

```diff
@@ -44,14 +44,18 @@ function createCoverColumn(config) {
   const titleH6 = document.createElement('h6');
   titleH6.classList.add('red');
   if (!config.isSubNav) {
-    const title = getMetadata('og:title').toUpperCase();
+    // page
+    const title = config.title ?? getMetadata('og:title');
```

Comment:
|| is needed here instead of ??  if the  config.title is empty,   `getMetadata('og:title')` is intended. 

?? (nullish coalescing operator) will only use the default value if the initial value is **null or undefined**.

Reply:
@rrusher Amit is right, if config.title is empty, `??` will keep that empty value. I think `||` is better here. 
See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing